Assignment 5.Rmd

```diff
@@ -99,13 +98,13 @@ The following results show that the predicted "no action" is less than the one o
 # Intervene: 2%
 
 # Predicted
-D3 <- apply(D2$prediction,1,which.max)
+
 # No Action
-sum(D3 == 3) / length(D3) # 64.5%
+sum(D2$prediction == "No Action") / nrow(D2) # 64.5%
 # Monitor Progress
-sum(D3 == 2) / length(D3) # 34.5%
+sum(D2$prediction == "Monitor Progress") / nrow(D2) # 34.5%
 # Intervene
-sum(D3 == 1) / length(D3) # 0.01%
+sum(D2$prediction == "Intervene") / nrow(D2) # 0.01%
```

Comment:
Good work in general! I am not sure why you need to add "class" in "predict" function. Sometimes it is good to specify the argument and add comments to explain that. Attach the predict function document here for your reference: https://www.rdocumentation.org/packages/stats/versions/3.6.0/topics/predict.lm